src/main/java/edu/cornell/kfs/concur/batch/service/ConcurStandardAccountingExtractReportService.java

```diff
@@ -0,0 +1,11 @@
+package edu.cornell.kfs.concur.batch.service;
+
+import java.io.File;
+
+import edu.cornell.kfs.concur.batch.report.ConcurStandardAccountingExtractBatchReportData;
+
+public interface ConcurStandardAccountingExtractReportService {
+    
+    File generateSummaryReport(ConcurStandardAccountingExtractBatchReportData reportData);
+
```

Comment:
I will be renaming this method to generateReport since the summary report is a subsection of the overall report.